db/index.js

```diff
@@ -0,0 +1,244 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/review');
+
+//create a review schema
+let ReviewSchema = mongoose.Schema({
+    //reviewId : {type: Number, unique: true},
+    houseId : Number,
+    customerName : String, 
+    customerProfilePhotoUrl : String,
+    customerReview : String,
+    createdAt : Date,
+    ratingCleanliness : Number,
+    ratingCheckIn : Number,
+    ratingLocation : Number,
+    ratingValue : Number
+});
+
+let Review = mongoose.model('Review', ReviewSchema);
+
+//save reviews in db
+let save = function(sampleReviews, callback) {
+    var sampleReviewsObject = sampleReviews.map (function(review) {
```

Comment:
try to use more lets/const instead of var!

Also, is the sampleReviewObj an array of objects? If so, maybe it can be renamed sampleReviewObjects to make that clearer.